app/scripts/modules/amazon/src/domain/IAmazonFunction.ts

```diff
@@ -29,6 +29,7 @@ export interface IAmazonFunction extends IFunction {
     subnetIds: [];
     vpcId: string;
   };
+  targetGroups: string | string[];
```

Comment:
Why is this typed as`string | string[]`?  Why not just type these as `string[]`?

Reply:
Currently we support adding only one target group instead of a list of target groups. Clouddriver expects it to be a string in Create/Update operation.
However, while fetching a function's details, AWS Lambda returns a list of string for the `targetGroups` field. 
Hence the type `string | string[]`